src/components/liveEventLink.tsx

```diff
@@ -0,0 +1,105 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { neutral, remSpace } from "@guardian/src-foundations";
+import { basePx, darkModeCss } from "styles";
+import { textSans, headline } from "@guardian/src-foundations/typography";
+import React, { ReactElement } from "react";
+import { SvgGuardianLiveLogo } from '@guardian/src-brand'
+
+const richLinkWidth = '8.75rem';
+
+type LiveEventLinkProps = {
+    url: string;
+    linkText: string;
+    price?: string;
+    image?: string;
+    start?: string;
+}
+
+const liveEventLinkStyles: SerializedStyles =
+    css`
+        background: ${neutral[97]};
+
+        a {
+            display: inline-block;
+            text-decoration: none;
+            color: ${neutral[7]};
+
+            div {
+                background: #B84376;
```

Comment:
Does this exist in any of the Source colour palettes?

Reply:
I don't think it does. The closest I can find is lifestyle.400 but I don't think there's a connection there.

I wonder if this colour is used elsewhere and should be added to the design system? @benwuersching @SiAdcock

Reply:
If it's the live events brand colour does it exist in a RR palette somewhere?